### PR TITLE
Insert raw auction request hooks invocation logic 

### PR DESCRIPTION
**Pull request by DGarbar**

### CODE REVIEW

**Review by rpanchyk — APPROVED** [review]

File: src/main/java/org/prebid/server/auction/AuctionRequestFactory.java [hunk 1]
```diff
@@ -231,11 +230,99 @@ private HttpRequestWrapper toHttpRequest(HookStageExecutionResult<EntrypointPayl
...
+            final String referer = paramsExtractor.refererFrom(httpRequest);
+            ortbTypesResolver.normalizeBidRequest(bidRequestNode, errors, referer);
+
+            return Future.succeededFuture(jsonNodeAsBidRequest(bidRequestNode));
+        } catch (Exception e) {
+            return Future.failedFuture(e);
+        }
+    }
+
+    private Future<AuctionContext> extractAccountAndExecuteRawAuctionHooks(BidRequest bidRequest,
```
Comment (schernysh) [inline src/main/java/org/prebid/server/auction/AuctionRequestFactory.java:246]: Method name is a bit misleading - it does much more than just extracts account and executes hooks.

File: src/main/java/org/prebid/server/auction/AuctionRequestFactory.java [hunk 2]
```diff
@@ -231,11 +230,99 @@ private HttpRequestWrapper toHttpRequest(HookStageExecutionResult<EntrypointPayl
...
+                .compose(account -> executeRawAuctionRequestHooksAndUpdate(
+                        bidRequest,
+                        httpRequestWrapper,
+                        account,
+                        hookExecutionContext,
+                        errors,
+                        timeout));
+    }
+
+    private Future<AuctionContext> executeRawAuctionRequestHooksAndUpdate(BidRequest bidRequest,
```
Comment (schernysh) [inline src/main/java/org/prebid/server/auction/AuctionRequestFactory.java:262]: This method name is a bit misleading too - `Update` suffix is too generic and forces to look into the body to find out what it's doing.

File: src/main/java/org/prebid/server/auction/AuctionRequestFactory.java (hunk 1 above)
Comment (DGarbar) [reply src/main/java/org/prebid/server/auction/AuctionRequestFactory.java]: renamed as executeRawAuctionHooksAndCreateAuctionContext

File: src/main/java/org/prebid/server/auction/AuctionRequestFactory.java (hunk 2 above)
Comment (DGarbar) [reply src/main/java/org/prebid/server/auction/AuctionRequestFactory.java]: Remove this to not duplicate such a long name

**Review by rpanchyk — APPROVED** [review]

**Pull request merged**